src/features/utils.ts

```diff
@@ -0,0 +1,68 @@
+import { window } from 'vscode';
+
+export const ALWAYS_ALLOW = 'Always Allow installs';
+export const ALWAYS_ASK = 'Always Ask before installs';
+export const INSTALL_NO_CONFIGURE = 'Install without configuring';
+
+export const YES_INSTALL = 'Yes, Install';
+export const NO_INSTALL = 'Do Not Install';
+
+export function promptForInstallPermissions(extensionName: string, packages: string): Thenable<string | undefined> {
```

Comment:
@karthiknadig what is the best way to handle all the string literals between the prompts and the decision cases? Is constants the right way?

Reply:
These should be put in `localize.ts`

Reply:
sounds good, I will move them over once we determine what they should say 

cc @cwebster-99 @luabud